app/enterprise/1.3-x/developer-portal/structure-and-file-types.md

```diff
@@ -182,20 +202,35 @@ Each theme file is compromised of a few different folders:
 - **theme.conf.yaml**
   - This config file sets color and font defaults available to templates for reference as css variables.  It also determines what options are available in the Kong Manager Appearance page.
 
-#### assets
-The asset folder contains css/js/fonts/images for your templates to reference.  To access asset files from your templates, keep in mind that Kong assumes a path from the root of your selected theme.
+#### Theme Assets
+
+##### Path
+- **format:** `theme/*/assets/**/*`
+
+##### Description
+The asset folder contains css/js/fonts/images for your templates to reference.
+
+To access asset files from your templates, keep in mind that Kong assumes a path from the root of your selected theme.
 
-- Examples:
-  - `themes/light-theme/assets/images/image1.jpeg` -> `<img src="assets/images/image1">`
-  - `themes/light-theme/assets/js/my-script.js` -> `<script src="assets/js/my-script.js"></script>`
-  - `themes/light-theme/assets/styles/my-styles.css` -> `<link href="assets/styles/normalize.min.css" rel="stylesheet" />`
+| Asset Path | Href Element |
+|--------------------------|---------------------------|
+| `themes/light-theme/assets/images/image1.jpeg` | `<img src="assets/images/image1">` |
+| `themes/light-theme/assets/js/my-script.js` | `<script src="assets/js/my-script.js"></script>` |
+| `themes/light-theme/assets/styles/my-styles.css` | `<link href="assets/styles/normalize.min.css" rel="stylesheet" />` |
 
-#### layouts
+#### Theme Layouts
+
+##### Path
+- **format:** `theme/*/layouts/**/*`
+- **file extensions:** `.html`
+
+##### Description
 Layouts act as the html skeleton of the page you wish to render.  Each file within the layouts directory must have an `html` filetype.  They can exist as vanilla `html`, or can reference partials and parent layouts via the portals templating syntax.  Layouts also have access to the `headmatter` and `body` attributes set in `content`.
 
 The example below shows what a typical layout could look like.
```

Comment:
I think we should add link to the other doc where we explain more about lua resty template syntax to explain {% %} and {* *}